modules/gdu/hm.nix

```diff
@@ -0,0 +1,35 @@
+{
+  mkTarget,
+  ...
+}:
+mkTarget {
+  name = "gdu";
+  humanName = "go DiskUsage()";
+
+  configElements = [
+    (
+      { colors }:
+      {
+        xdg.configFile."gdu/gdu.yaml" = {
+          enable = true;
+          text = with colors.withHashtag; ''
```

Comment:
This seems hard to read in light mode (`nix run .#testbed:gdu:light`):

<img width="660" height="486" alt="Image" src="https://github.com/user-attachments/assets/afeb469a-5a30-455b-88cb-9d5421539f8d" />

Is this intentional?